plutus-pab/src/Plutus/PAB/Monitoring.hs

```diff
@@ -106,6 +107,14 @@ toSeverity = \case
   L.Alert     -> Alert
   L.Emergency -> Emergency
 
+handleLogEffects ::
```

Comment:
The name maybe isn't the best? Open for suggestions.

Reply:
maybe `runLogTrace` ? I tend to use `handle` for functions that remove one or more of the effects but still return an `Eff effs`